Create ui-c8y-1023-14-33-prevent-plugin-translations-from-overriding-app-translations-10745-graft-release-y.md (#4092)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-14-33-prevent-plugin-translations-from-overriding-app-translations-10745-graft-release-y.md

---
date: ""

> title: Prevent plugin translations from overriding app translations

Reviewer: ```suggestion
title: Prevented plugin translations from overriding application translations
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65724
version: 1023.14.33
---
This fix prevents plugins from overriding the translations in the shell application. Still, within a plugin, the translations provided by this plugin take precedence over the shell application's ones.

> If you need to override a shell application translation, you may use [application options](https://cumulocity.com/docs/web/application-configuration/#languages-customization) or [localization feature](https://cumulocity.com/docs/standard-tenant/changing-settings/#localization).

Reviewer: ```suggestion
This fix prevents plugins from overriding the translations in the shell application. Still, within a plugin, the translations provided by this plugin take precedence over the shell application's ones.
If you want to override a shell application translation, you may use [application options](https://cumulocity.com/docs/web/application-configuration/#languages-customization) or [the localization feature](https://cumulocity.com/docs/standard-tenant/changing-settings/#localization).
```

> If you want to override a shell application translation, you may use [application options](https://cumulocity.com/docs/web/application-configuration/#languages-customization) or [the localization feature](https://cumulocity.com/docs/standard-tenant/changing-settings/#localization).

Reviewer 2: ```suggestion
If you want to override a shell application translation, you may use [application options](/web/application-configuration/#languages-customization) or [the localization feature](/standard-tenant/changing-settings/#localization).
```

---
date: ""

> title: Prevent plugin translations from overriding app translations (#10745) [GRAFT][release/y2026] (#10768)

Reviewer: ```suggestion
title: Prevent plugin translations from overriding app translations
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65724
version: 1023.14.33
---
If language files from application and plugin contain the same key, the
translation from plugin overrides the translation from app, which is not
desired because the translation in the app is usually newer, more
relevant and up-to-date than a translation from plugin.

This task is to change the order of loading translations, so that
translations from app have higher priority than translations from

> plugins.

Reviewer: ```suggestion
This fix prevents plugins from overriding the translations in the shell application. Still, within a plugin, the translations provided by this plugin take precedence over the shell application's ones.
If you need to override a shell application translation, you may use [application options](https://cumulocity.com/docs/web/application-configuration/#languages-customization) or [localization feature](https://cumulocity.com/docs/standard-tenant/changing-settings/#localization).
```
